src/bsoncxx/builder/stream/single_context.hpp

```diff
@@ -79,11 +82,19 @@ class single_context {
     }
 
    private:
+    BSONCXX_INLINE array_context<single_context> wrap_array() {
```

Comment:
I'd just as soon leave them out.  There's not much value to a maintainer or implementer beyond what the class comment indicates, and as private methods they don't need generated docs